src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -680,24 +680,24 @@ private IntPtr ObjectToHandle(object obj)
             return handle;
         }
 
-        private object HandleToObject(IntPtr handle)
+        private object HandleToObject(void* handle)
         {
 #if DEBUG
-            handle = new IntPtr(~(long)s_handleHighBitSet & (long) handle);
+            handle = (void*)(~(long)s_handleHighBitSet & (long) handle);
```

Comment:
```suggestion
            handle = (void*)(~s_handleHighBitSet & (nint)handle);
```

Reply:
`s_handleHighBitSet`  is IntPtr so it warns cannot use ~ operator

Reply:
Ok, does `handle = (void*)(~(nuint)s_handleHighBitSet & (nuint)handle);` work?

Reply:
`CS0030: Cannot convert type 'System.IntPtr' to 'nuint'`